src/test/java/com/facebook/presto/hadoop/TestLineReader.java

```diff
@@ -87,6 +88,7 @@ public void testDefaultReaderMaxBytesConsumed()
             fail("Expected IOException");
         }
         catch (IOException e) {
+            assert (e instanceof MaxTextLineLengthExceededIOException);
```

Comment:
No need to check the instance. Just change catch clause to only catch this exception.